frontend/src/layout/navigation/TopBar/announcementLogic.ts

```diff
@@ -16,6 +17,9 @@ export enum AnnouncementType {
     AttentionRequired = 'AttentionRequired',
 }
 
+export const DefaultCloudAnnouncement =
```

Comment:
```suggestion
export const DEFAULT_CLOUD_ANNOUNCEMENT =
```
so that this looks like a JS constant rather than a class or interface